ucp/benchmarks/utils.py

```diff
@@ -22,6 +22,43 @@ def _ensure_cuda_device(devs, rank):
     numba.cuda.current_context()
 
 
+def get_allocator(
+    object_type: str, rmm_init_pool_size: int, rmm_managed_memory: bool
+) -> object:
+    """
+    Initialize and return array-allocator based on arguments passed.
+
+    Parameters
+    ----------
+    object_type: str
+        The type of object the allocator should return. Options are: "numpy", "cupy"
+        or "rmm".
+    rmm_init_pool_size: int
+        If the object type is "rmm" (implies usage of RMM pool), define the initial
+        pool size.
+    rmm_managed_memory: bool
+        If the object type is "rmm", use managed memory if `True`, or default memory
+        otherwise.
+    """
+    if object_type == "numpy":
+        import numpy as xp
+    elif object_type == "cupy":
+        import cupy as xp
+    else:
+        import cupy as xp
+
+        import rmm
```

Comment:
ditto

Reply:
They could, but we need to test for their availability anyway, for example, a UCX-Py user may not have any CUDA libraries and importing `cupy` or `rmm` would then fail, or just `rmm` may fail if it's not a RAPIDS user. Therefore, I would have to import at the top and still handle the cases here when the import was unsuccessful, which will probably making the code even more complex, so I would prefer to keep them like this. What do you think?